content/collections/data/en/amplitude-shopify-plugin.md

```diff
@@ -0,0 +1,48 @@
+---
+id: d7658703-c1c3-4cc4-a5e3-3b5674e41405
+blueprint: data
+title: 'Amplitude Shopify Plugin'
+landing: false
+exclude_from_sitemap: false
+updated_by: 0c3a318b-936a-4cbd-8fdf-771a90c297f0
+updated_at: 1729791421
+---
+The [Amplitude Shopify Plugin](https://apps.shopify.com/amplitude) enables you to instrument your Shopify store with an Amplitude features including [Autocapture](/docs/data/autocapture) and [Session Replay](/docs/session-replay).
+
```

Comment:
### About Shopify
[Shopify](https://www.shopify.com/) is an all-in-one commerce platform that allows businesses of any size to create, customize, and manage online stores with ease. It offers tools for product listings, payments, shipping, and customer engagement, streamlining the selling process online, across social media, and in person. With built-in marketing features, analytics, and integrations with third-party apps, Shopify helps business owners track performance and drive sales.

Reply:
### Considerations

- **Performance**: The packages we utilize on the frontend is roughly 167kb. To give a rough idea of the performance impact I generated 6 different Lighthouse Performance reports. 3 with the Amplitude packages, and 3 without. Without the Amplitude packages, the average performance score was 98. With the packages, the average Performance score was 96.
- **Session replay may not display as expected**. Shopify’s UI heavily relies on animations, specifically the fade-in animation. The animation responsible for changing the opacity of the UI from 0 to 1 is not consistently picked up by session replay, which is why elements appear to be missing.